RAPIDS.cmake

```diff
@@ -1,5 +1,5 @@
 #=============================================================================
-# Copyright (c) 2021, NVIDIA CORPORATION.
+# Copyright (c) 2021-2023, NVIDIA CORPORATION.
```

Comment:
These files have been modified in 2023 but for some reason their copyright headers have not been kept up to date and CI didn't fail. I'm lumping these copyright year changes into this PR because pre-commit fails locally without these changes.